src/app/models/AttributeDecorators.ts

```diff
@@ -60,25 +61,45 @@ export function bawImage<Model>(
     const imageASize = a.height * a.width;
     const imageBSize = b.height * b.width;
     return imageASize === imageBSize ? 0 : imageASize > imageBSize ? -1 : 1;
-  }
+  };
 
   function missingDefault(images: ImageUrl[]): boolean {
     return !images.find((image) => image.size === ImageSizes.default);
   }
 
+  function sanitizeUrl(apiRoot: string, url: string): string {
```

Comment:
nit: this is more normalization than sanitization (cleaning for malicious values)